mon-pix/app/authenticators/oidc.js

```diff
@@ -74,7 +60,8 @@ export default class PoleEmploiAuthenticator extends BaseAuthenticator {
       id_token: data.id_token,
       source: decodedAccessToken.source,
       user_id: decodedAccessToken.user_id,
-      identity_provider: decodedAccessToken.identity_provider,
+      // for backwards compatibility TODO remove after a couple days in production
+      identity_provider_code: decodedAccessToken.identity_provider || decodedAccessToken.identity_provider_code,
```

Comment:
**Remarque** : L'API ne retourne pas `identity_provider_code`, pourquoi faire `|| decodedAccessToken.identity_provider_code,` ici ?

Reply:
parce que ca va venir, c'est pour anticipe le changement sans rien casse

Reply:
sauf si vous preferez que je fasse la modif dans cette PR ?

Reply:
parce qu'on veut le renommer comme ça coté back à terme c'est ça ? 
Autant le traiter au moment où on touche au back plutôt.

Reply:
le terme `code` me paraît un peu bizarre 🤔 
c'est pas plutôt `key` qu'on voudrait mettre ? (ou peut être à discuter dans le PR côté back ?)

Reply:
pour moi c'est pas un cle, une cle c'est qq ch qui nous permet d'extraire une valeur d'une map ou d'un objet json, et je prefererai qu'on decide la, parce que l'idee c'est d'harmoniser le nom back et front, donc si code ne vous va pas, faut que je le change dans le front avant de merger.. j'ai mis code parce qu'en anglais c'est defini par "a system of words, letters, figures, or symbols used to represent others" et c'est ca dans ce cas, c'est ce qu'on utilise dans la BDD pour identifier l'identity provider